htdocs/web_portal/views/user/view_user.php

```diff
@@ -72,28 +72,38 @@
             </ul>
         </div>
 
-
+        <?php
+        $isUserAdmin = ($params['user'] && $params['user']->isAdmin());
+        ?>
         <!--  User -->
         <div class="tableContainer" style="width: 55%; float: left;">
             <span class="header" style="vertical-align:middle; float: left; padding-top: 0.9em; padding-left: 1em;">
                 User Details
             </span>
             <img src="<?php echo \GocContextPath::getPath()?>img/contact_card.png" class="decoration" />
             <table style="clear: both; width: 100%; table-layout: fixed;">
-                <tr class="site_table_row_1">
+            <?php if ($isUserAdmin) : ?>
+                <tr class="site_table_row_even">
+                    <td class="site_table">Last Logged In</td>
+                    <td class="site_table">
+                        <?php xecho($params['user']->getLastLoginDate()->format('Y-m-d H:i:s')) ?>
```

Comment:
weirdly, this only seems to display for users with a LastLogin time...

![image](https://github.com/GOCDB/gocdb/assets/2488355/48adea61-bd32-4311-a6f3-8dee3d8780f6)

This is a edge case, as the only users without a LastLogin time will be sample users on dev instances, so I'm not to bothered by it. I just can't figure out why it's happening. Any ideas?

Reply:
The fix, I have provided now should address this as we now show empty field if last logged time is NOT available.